kubernetes-model-generator/kubernetes-model-apiextensions/src/test/resources/valid-ConversionReview.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Could you please rename the file to `valid-conversionreview.json`(all smallcases) just like other files? 

Please fix the indentation in this file too.

Reply:
👍